app/forms/forms/submission_email_form.rb

```diff
@@ -2,7 +2,7 @@ class Forms::SubmissionEmailForm < BaseForm
   attr_accessor :form, :temporary_submission_email, :email_code, :confirmation_code, :current_user, :notify_response_id
 
   EMAIL_REGEX = /.*@.*/
-  GOVUK_EMAIL_REGEX = /\.gov\.uk\z/i
+  GOVUK_EMAIL_REGEX = /(\.gov\.uk|pentestpartners\.com)\z/i # TODO: remove after Friday 3 Nov 2023
```

Comment:
Its a pity we couldn't use an env var to control access for both authentication and for this feature.

Reply:
hmmm that's a good point... I'll rework this

Reply:
We've done it a hacky way for now...